### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ryanwholey**
Update the readme to include inputs, outputs and examples

### CODE REVIEW

**Review by bendrucker — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,134 @@
...
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: takescoop/terraform-cloud-workspace-action@v0
+        with:
+          terraform_token: "${{ secrets.TF_TOKEN }}"
+          terraform_organization: "${{ takescoop }}"
+          apply: "${{ github.ref == 'refs/heads/main' }}"
```
Comment (bendrucker) [inline README.md:18]: https://stackoverflow.com/a/68414395

Comment (bendrucker) [reply README.md:18]:
```suggestion
          apply: "${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}"
```

Comment (bendrucker) [reply README.md:18]: Try it to be sure. We also have `$default-branch` to use in workflow templates, alternative would just be to use that syntax in this example and have a code comment telling the user to replace it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,134 @@
...
+name: TFLint
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: takescoop/terraform-cloud-workspace-action@v0
+        with:
+          terraform_token: "${{ secrets.TF_TOKEN }}"
+          terraform_organization: "${{ takescoop }}"
```
Comment (bendrucker) [inline README.md:17]:
```suggestion
          terraform_organization: "my-org"
```

Comment (ryanwholey) [reply README.md:17]: whoops!

File: README.md (hunk 1 above)
Comment (ryanwholey) [reply README.md]: it works! 🥳

**Review by bendrucker — APPROVED** [review]

**Pull request merged**